### PR TITLE
Address unused XEUS_R_EMSCRIPTEN_WASM_BUILD variable

### PR DESCRIPTION
**Pull request by anutosh491**
Continued on top of #105

### CODE REVIEW

**Comment by anutosh491** [comment]
Minor refactor. This should be ready.

File: share/jupyter/kernels/xr/kernel.json.in [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "display_name": "R 4.3.1 (xr)",
+  "display_name": "R 4.4.1 (xr)",
```
Comment (IsabelParedes) [inline share/jupyter/kernels/xr/kernel.json.in:2]: Is it possible to change this to reflect the version installed in the environment? Something like
```json
"display_name": "R @R_VERSION_MAJOR@.@R_VERSION_MINOR@ (xr)",
```

**Review by IsabelParedes — CHANGES_REQUESTED** [review]
Thanks @anutosh491!
Could you also add the `--no-readline` argument when creating the interpreter to pass it to `RF_initEmbeddedR()` but only when building with emscripten? And probably just add a small note "`r-base` is not compiled with `readline`"

File: share/jupyter/kernels/xr/kernel.json.in (hunk 1 above)
Comment (anutosh491) [reply share/jupyter/kernels/xr/kernel.json.in:2]: Sure !

**Comment by anutosh491** [comment]
Hey @IsabelParedes made the changes. Hopefully this is all what we wanted !

**Review by IsabelParedes — APPROVED** [review]

**Pull request merged**